xypath/loader.py

```diff
@@ -0,0 +1,31 @@
+import messytables
+import xypath
+
+def table_set(filename):
```

Comment:
`table_set` does indeed return all the tables.
But then how do you get a subset of those tables? That's `get_sheets` job. Writing docstrings.